### PR TITLE
Add note about explicit selection of non-aggregate reports

### PR DESCRIPTION
**Pull request by Godin**
As discussed in #418

### CODE REVIEW

**Comment by Godin** [comment]
@marchof could you please verify? Note that I decided to update "usage" section rather than specific "report-aggregate" goal, think this is better.

File: org.jacoco.doc/docroot/doc/maven.html [hunk 1]
```diff
@@ -103,6 +103,33 @@
 mvn help:describe -Dplugin=org.jacoco:jacoco-maven-plugin -Ddetail
 </pre>
 
+<p>
+  Usage of plugin in reporting section without
+  <a href="https://maven.apache.org/plugins/maven-site-plugin/examples/configuring-reports.html#Selecting_Reports_from_a_Plugin:_Configuring_Report_Sets">explicit selection of reports</a>
+  might lead to generation of redundant aggregate reports.
```
Comment (marchof) [inline org.jacoco.doc/docroot/doc/maven.html]: Only when using the site goal, right?

**Comment by marchof** [comment]
@Godin Beside mention of site looks good to me!

File: org.jacoco.doc/docroot/doc/maven.html (hunk 1 above)
Comment (Godin) [inline org.jacoco.doc/docroot/doc/maven.html]: @marchof right, fixed

**Pull request merged**